Remove "charge" transactions from refund process (#903)
commercetools/commercetools-adyen-integration: extension/docs/Refund.md

#### Prerequisites

> It is required that the payment has one `Authorized` transaction with state `Success`.

Reviewer: ```suggestion
It is required that the payment has a transaction of type `Authorization` and state `Success`.
```
